modules/manually-create-identity-access-management.adoc

```diff
@@ -228,6 +228,24 @@ spec:
 +
 endif::ash[] 
 
+.Sample `Secret` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-image-registry
+  name: installer-cloud-credentials
+data:
+  azure_subscription_id: <base64-encrypted-azure-subscription-id>
+  azure_client_id: <base64-encrypted-azure-client-id>
+  azure_client_secret: <base64-encrypted-azure-client-secret>
+  azure_tenant_id: <base64-encrypted-azure-tenant-id>
+  azure_resource_prefix: <base64-encrypted-azure-resource-prefix>
+  azure_resourcegroup: <base64-encrypted-azure-resourcegroup>
+  azure_region: <base64-encrypted-azure-region>
+----
++
```

Comment:
sorry, I have one more comment here.
[Line204 - line226](https://github.com/openshift/openshift-docs/pull/55910/files#diff-aba65666970ec0397d998aa5b6083c7ea6e71a39ec5b3a3de7e2192bdccf2f26R204-R226) is also applicable here, not only for ASH. The procedure to create secrets manually from Credential objects are the same on both azure and azure stack hub. In order to keep consistent with ash part, could you also give example of CredentialsRequest object here?

Reply:
No worries- Addressed

Reply:
thanks for the update. The content looks good to me.
Suggest to adjust the sequence of sample secrets and sample CrendentialsRequest, move secrets behind CrendentialsRequest ?

Reply:
Addressed

Reply:
thanks, looks good to me now.